uhabits-android/src/main/java/org/isoron/uhabits/activities/habits/show/views/ScoreCardView.kt

```diff
@@ -36,7 +36,7 @@ class ScoreCardView(context: Context, attrs: AttributeSet) : LinearLayout(contex
         val androidColor = state.color.toThemedAndroidColor(context)
         binding.title.setTextColor(androidColor)
         binding.spinner.setSelection(state.spinnerPosition)
-        binding.scoreView.setScores(state.scores)
+        binding.scoreView.setScores(state.scores.toMutableList())
```

Comment:
If possible, please change the type of `ScoreChart.scores` to `List<Score>` instead of changing this line.